### ISSUE TITLE
Pre-work Checklist: Developer: Aidan Simmons

### ISSUE THREAD

**Issue by aidanwsimmons** [hackforla/website#5722]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Self Assign this issue (gear in right side panel)
- [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
- [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues in the [prioritized backlog](https://github.com/hackforla/website/projects/7#column-7198257) only with increasing complexity in the following order:
  - Good first issue (two per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by t-will-gillis** [comment]
Inactive member; closed due to inactivity

**Comment by aidanwsimmons** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
Took me about 45 minutes to get through the pre-work checklist up until this prompt

**Comment by HackforLABot** [comment]
## Activity Log: aidanwsimmons
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- aidanwsimmons opened: https://github.com/hackforla/website/issues/5722 at 10/16/2023, 07:38 PM PDT
- aidanwsimmons assigned: https://github.com/hackforla/website/issues/5722 at 10/16/2023, 07:51 PM PDT
- aidanwsimmons issue closed as completed: https://github.com/hackforla/website/issues/5722#event-11679252273 at 02/01/2024, 09:26 PM PST
- aidanwsimmons reopened: https://github.com/hackforla/website/issues/5722#event-11679252273 at 02/05/2024, 07:24 PM PST
- aidanwsimmons assigned: https://github.com/hackforla/website/issues/6053 at 02/09/2024, 02:50 PM PST
- aidanwsimmons assigned: https://github.com/hackforla/website/issues/6054 at 02/09/2024, 02:50 PM PST
- aidanwsimmons unassigned: https://github.com/hackforla/website/issues/6053 at 02/09/2024, 02:51 PM PST
- aidanwsimmons commented on: https://github.com/hackforla/website/issues/6054#issuecomment-1936720342 at 02/09/2024, 03:07 PM PST
- aidanwsimmons commented on: https://github.com/hackforla/website/issues/5722#issuecomment-1936721304 at 02/09/2024, 03:08 PM PST
- aidanwsimmons commented on: https://github.com/hackforla/website/issues/6054#issuecomment-1936722539 at 02/09/2024, 03:10 PM PST
- aidanwsimmons opened: https://github.com/hackforla/website/pull/6273 at 02/09/2024, 03:35 PM PST
- aidanwsimmons closed as completed: https://github.com/hackforla/website/issues/5722#event-11763331139 at 02/09/2024, 03:44 PM PST
- aidanwsimmons assigned: https://github.com/hackforla/website/issues/6053 at 02/10/2024, 02:50 PM PST
- aidanwsimmons unassigned: https://github.com/hackforla/website/issues/6053 at 02/10/2024, 02:55 PM PST
- aidanwsimmons commented on: https://github.com/hackforla/website/pull/6273#issuecomment-1937354976 at 02/10/2024, 03:41 PM PST
- aidanwsimmons assigned: https://github.com/hackforla/website/issues/5171 at 02/12/2024, 10:55 AM PST
- aidanwsimmons commented on: https://github.com/hackforla/website/issues/5171#issuecomment-1939357451 at 02/12/2024, 11:03 AM PST
- aidanwsimmons commented on: https://github.com/hackforla/website/issues/5171#issuecomment-1939368703 at 02/12/2024, 11:11 AM PST
- aidanwsimmons closed: https://github.com/hackforla/website/pull/6273#event-11778969500 at 02/12/2024, 11:32 AM PST
- aidanwsimmons opened: https://github.com/hackforla/website/pull/6282 at 02/12/2024, 12:10 PM PST
- aidanwsimmons commented on: https://github.com/hackforla/website/pull/6282#issuecomment-1939483844 at 02/12/2024, 12:11 PM PST
- aidanwsimmons unassigned: https://github.com/hackforla/website/issues/5171 at 02/12/2024, 12:12 PM PST
- aidanwsimmons commented on: https://github.com/hackforla/website/issues/5171#issuecomment-1939492244 at 02/12/2024, 12:13 PM PST
- aidanwsimmons PR was merged: https://github.com/hackforla/website/pull/6282#event-11780010005 at 02/12/2024, 01:21 PM PST
- aidanwsimmons issue closed by PR 6282: https://github.com/hackforla/website/issues/6054#event-11780010191 at 02/12/2024, 01:21 PM PST
- aidanwsimmons assigned: https://github.com/hackforla/website/issues/5178 at 02/12/2024, 04:17 PM PST
- aidanwsimmons commented on: https://github.com/hackforla/website/issues/5178#issuecomment-1939858302 at 02/12/2024, 04:19 PM PST
- aidanwsimmons opened: https://github.com/hackforla/website/pull/6283 at 02/12/2024, 04:35 PM PST
- aidanwsimmons commented on: https://github.com/hackforla/website/pull/6281#issuecomment-1939890101 at 02/12/2024, 04:37 PM PST
- aidanwsimmons submitted review: https://github.com/hackforla/website/pull/6281#pullrequestreview-1876646950 at 02/12/2024, 04:43 PM PST
- aidanwsimmons commented on: https://github.com/hackforla/website/pull/6281#issuecomment-1939910208 at 02/12/2024, 04:43 PM PST
- aidanwsimmons commented on: https://github.com/hackforla/website/pull/6283#issuecomment-1940613719 at 02/12/2024, 11:42 PM PST
- aidanwsimmons commented on: https://github.com/hackforla/website/pull/6279#issuecomment-1943034010 at 02/13/2024, 07:36 PM PST
- aidanwsimmons submitted review: https://github.com/hackforla/website/pull/6279#pullrequestreview-1881588551 at 02/14/2024, 05:20 PM PST
- aidanwsimmons PR was merged: https://github.com/hackforla/website/pull/6283#event-11811037717 at 02/14/2024, 10:53 PM PST
- aidanwsimmons commented on: https://github.com/hackforla/website/issues/5178#issuecomment-1967731895 at 02/27/2024, 02:16 PM PST
- aidanwsimmons unassigned: https://github.com/hackforla/website/issues/5178 at 02/27/2024, 03:54 PM PST
- aidanwsimmons assigned: https://github.com/hackforla/website/issues/6375 at 02/27/2024, 04:00 PM PST
- aidanwsimmons commented on: https://github.com/hackforla/website/issues/6375#issuecomment-1967944565 at 02/27/2024, 04:09 PM PST
- aidanwsimmons opened: https://github.com/hackforla/website/pull/6386 at 02/27/2024, 04:37 PM PST
- aidanwsimmons commented on: https://github.com/hackforla/website/pull/6384#issuecomment-1969563986 at 02/28/2024, 10:12 AM PST
- aidanwsimmons commented on: https://github.com/hackforla/website/pull/6362#issuecomment-1969564548 at 02/28/2024, 10:12 AM PST
- aidanwsimmons submitted review: https://github.com/hackforla/website/pull/6362#pullrequestreview-1907642805 at 02/28/2024, 03:29 PM PST
- aidanwsimmons submitted review: https://github.com/hackforla/website/pull/6384#pullrequestreview-1907679194 at 02/28/2024, 04:02 PM PST
- aidanwsimmons commented on: https://github.com/hackforla/website/pull/6386#issuecomment-1974135587 at 03/01/2024, 04:38 PM PST
- aidanwsimmons PR was merged: https://github.com/hackforla/website/pull/6386#event-11992263633 at 03/03/2024, 05:49 PM PST
- aidanwsimmons issue closed by PR 6386: https://github.com/hackforla/website/issues/6375#event-11992263742 at 03/03/2024, 05:49 PM PST
- aidanwsimmons assigned: https://github.com/hackforla/website/issues/5843 at 03/04/2024, 06:04 PM PST
- aidanwsimmons commented on: https://github.com/hackforla/website/issues/5843#issuecomment-1977815356 at 03/04/2024, 06:05 PM PST
- aidanwsimmons opened: https://github.com/hackforla/website/pull/6417 at 03/04/2024, 06:27 PM PST
- aidanwsimmons PR was merged: https://github.com/hackforla/website/pull/6417#event-12035475518 at 03/06/2024, 05:44 PM PST
- aidanwsimmons issue closed by PR 6417: https://github.com/hackforla/website/issues/5843#event-12035475607 at 03/06/2024, 05:44 PM PST
- aidanwsimmons assigned: https://github.com/hackforla/website/issues/5997 at 03/07/2024, 02:34 PM PST
- aidanwsimmons commented on: https://github.com/hackforla/website/issues/5997#issuecomment-1984667493 at 03/07/2024, 02:35 PM PST
- aidanwsimmons issue closed as completed: https://github.com/hackforla/website/issues/5997#event-12191807256 at 03/20/2024, 02:51 PM PDT
- aidanwsimmons assigned: https://github.com/hackforla/website/issues/6510 at 03/25/2024, 12:22 PM PDT
- aidanwsimmons commented on: https://github.com/hackforla/website/issues/6510#issuecomment-2018743611 at 03/25/2024, 12:22 PM PDT
- aidanwsimmons opened: https://github.com/hackforla/website/pull/6521 at 03/26/2024, 05:17 PM PDT
- aidanwsimmons commented on: https://github.com/hackforla/website/pull/6518#issuecomment-2023325054 at 03/27/2024, 10:03 AM PDT
- aidanwsimmons commented on: https://github.com/hackforla/website/pull/6503#issuecomment-2023325451 at 03/27/2024, 10:04 AM PDT
- aidanwsimmons submitted review: https://github.com/hackforla/website/pull/6518#pullrequestreview-1964628762 at 03/27/2024, 01:01 PM PDT
- aidanwsimmons PR was merged: https://github.com/hackforla/website/pull/6521#event-12302225921 at 03/30/2024, 08:11 PM PDT
- aidanwsimmons issue closed by PR 6521: https://github.com/hackforla/website/issues/6510#event-12302225950 at 03/30/2024, 08:11 PM PDT
- aidanwsimmons submitted review: https://github.com/hackforla/website/pull/6566#pullrequestreview-1981218047 at 04/04/2024, 02:28 PM PDT
- aidanwsimmons commented on: https://github.com/hackforla/website/pull/6566#issuecomment-2038262081 at 04/04/2024, 02:29 PM PDT
- aidanwsimmons submitted review: https://github.com/hackforla/website/pull/6566#pullrequestreview-1990656273 at 04/09/2024, 07:25 PM PDT
- aidanwsimmons assigned: https://github.com/hackforla/website/issues/6622 at 04/12/2024, 01:03 PM PDT
- aidanwsimmons commented on: https://github.com/hackforla/website/issues/6622#issuecomment-2052450947 at 04/12/2024, 01:04 PM PDT
- aidanwsimmons commented on: https://github.com/hackforla/website/issues/6622#issuecomment-2057678507 at 04/15/2024, 12:45 PM PDT
- aidanwsimmons opened: https://github.com/hackforla/website/pull/6651 at 04/15/2024, 01:07 PM PDT
- aidanwsimmons PR was merged: https://github.com/hackforla/website/pull/6651#event-12551509072 at 04/21/2024, 10:37 AM PDT
- aidanwsimmons issue closed by PR 6651: https://github.com/hackforla/website/issues/6622#event-12551509117 at 04/21/2024, 10:37 AM PDT
- aidanwsimmons commented on: https://github.com/hackforla/website/pull/6708#issuecomment-2068279223 at 04/21/2024, 05:14 PM PDT
- aidanwsimmons submitted review: https://github.com/hackforla/website/pull/6708#pullrequestreview-2015864269 at 04/22/2024, 02:53 PM PDT
- aidanwsimmons commented on: https://github.com/hackforla/website/pull/6712#issuecomment-2075469497 at 04/24/2024, 10:26 AM PDT
- aidanwsimmons submitted review: https://github.com/hackforla/website/pull/6712#pullrequestreview-2020867771 at 04/24/2024, 01:18 PM PDT
- aidanwsimmons assigned: https://github.com/hackforla/website/issues/6548 at 04/24/2024, 04:11 PM PDT
- aidanwsimmons commented on: https://github.com/hackforla/website/issues/6548#issuecomment-2076013026 at 04/24/2024, 04:12 PM PDT
- aidanwsimmons opened: https://github.com/hackforla/website/pull/6816 at 05/03/2024, 06:05 PM PDT
- aidanwsimmons closed: https://github.com/hackforla/website/pull/6816#event-12736899416 at 05/07/2024, 05:51 PM PDT
- aidanwsimmons commented on: https://github.com/hackforla/website/issues/6548#issuecomment-2103820897 at 05/09/2024, 09:04 PM PDT
- aidanwsimmons commented on: https://github.com/hackforla/website/issues/6548#issuecomment-2106501961 at 05/12/2024, 06:59 PM PDT
- aidanwsimmons unassigned: https://github.com/hackforla/website/issues/6548 at 05/12/2024, 06:59 PM PDT
- aidanwsimmons submitted review: https://github.com/hackforla/website/pull/6887#pullrequestreview-2084624621 at 05/29/2024, 01:06 AM PDT
- aidanwsimmons assigned: https://github.com/hackforla/website/issues/6910 at 06/06/2024, 03:34 PM PDT
- aidanwsimmons commented on: https://github.com/hackforla/website/issues/6910#issuecomment-2153510126 at 06/06/2024, 03:35 PM PDT
- aidanwsimmons opened: https://github.com/hackforla/website/pull/6985 at 06/10/2024, 03:45 PM PDT
- aidanwsimmons commented on: https://github.com/hackforla/website/pull/6976#issuecomment-2162032534 at 06/11/2024, 08:25 PM PDT
- aidanwsimmons PR was merged: https://github.com/hackforla/website/pull/6985#event-13135052450 at 06/12/2024, 10:11 AM PDT
- aidanwsimmons issue closed by PR 6985: https://github.com/hackforla/website/issues/6910#event-13135052646 at 06/12/2024, 10:11 AM PDT
- aidanwsimmons submitted review: https://github.com/hackforla/website/pull/6976#pullrequestreview-2114582929 at 06/12/2024, 07:56 PM PDT
